elife/config/etc-init-goaws-init.conf

```diff
@@ -0,0 +1,11 @@
+description "GOAWS will run a local AWS server that can be used by setting --endpoint=http://localhost:4100 to commands"
+respawn
+respawn limit 10 30
+kill timeout 70
+setuid {{ pillar.elife.deploy_user.username }}
+env HOME=/home/{{ pillar.elife.deploy_user.username }}
```

Comment:
by default no environment variables are available in Upstart jobs, so all needed ones should be specified here